src/prx/main.py

```diff
@@ -199,26 +199,6 @@ def build_records(
     rinex_3_obs_file,
     rinex_3_ephemerides_files,
     approximate_receiver_ecef_position_m,
-):
```

Comment:
caching the result of prx's actual computation does not make sense, as this is where we typically develop, set breakpoints etc., so we want this code to run. Caching makes a lot of sense for calls to external code such as georinex parsing.